tobac/tests/test_feature_detection.py

```diff
@@ -1,4 +1,5 @@
 import tobac.testing
```

Comment:
```suggestion

```

This import can probably removed then?

Reply:
Yep, I've removed the extra imports and we now only import `tobac.testing` once.